README.md

```diff
@@ -61,6 +61,22 @@ From command-line:
 
 *Todo list here*
 
+## Design
+
+### Endpoints
```

Comment:
```suggestion
## Endpoints
```

so it will look like all of the other titles

Reply:
I wanted it to be a sub-title of design